client/packages/invoices/src/Prescriptions/api/hooks/usePrescription.ts

```diff
@@ -196,10 +197,12 @@ const useUpdate = (id: string) => {
   const { prescriptionApi, storeId, queryClient } = usePrescriptionGraphQL();
 
   const mutationFn = async (patch: RecordPatch<PrescriptionRowFragment>) => {
+    console.log('patch', patch);
     const input: UpdatePrescriptionInput = {
       ...patch,
       id,
       status: mapStatus(patch),
+      diagnosisId: setNullableInput('diagnosisId', patch),
```

Comment:
This is in two places, I wonder if we need to generalise this some how. Maybe even the draft patch state thing can do something special with Nullable Updates?